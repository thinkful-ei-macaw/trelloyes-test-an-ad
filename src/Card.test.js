import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Card from './Card.js';

// Smoke Test
describe('<Card />', () => {
    it('renders without crashing', () =>{
        const div = document.createElement('div');
        ReactDOM.render(<Card 
            title="Some title"
            content="A string of content" />, div);
        ReactDOM.unmountComponentAtNode(div);
    })
})

// Snapshot Tests
describe('<Card />', () => {
    it('render this UI as expected', () => {
        const tree = renderer.create(<Card 
            title="Some title"
            content="A string of content" />).toJSON();
        expect(tree).toMatchSnapshot();
    })
})

// title -> string
// content -> string

