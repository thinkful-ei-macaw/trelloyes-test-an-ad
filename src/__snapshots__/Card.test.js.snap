// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> render this UI as expected 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3>
    Some title
  </h3>
  <p>
    A string of content
  </p>
</div>
`;
